src/styles/mixins/_inputs.scss

```diff
@@ -0,0 +1,142 @@
+/* This file contains mixins with styles used in all types of inputs */
+
+/* Common elements */
+
+@mixin inputHelpText {
+  color: var(--kui-color-text-neutral, $kui-color-text-neutral);
+  font-family: var(--kui-font-family-text, $kui-font-family-text);
+  font-size: var(--kui-font-size-20, $kui-font-size-20);
+  font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
+  line-height: var(--kui-line-height-20, $kui-line-height-20);
+}
+
+/* Text inputs */
+
+@mixin inputDefaults {
+  -webkit-appearance: none; // need this to allow box-shadow to apply on mobile
+  background-color: var(--kui-color-background, $kui-color-background);
+  border: 0;
+  border-radius: var(--kui-border-radius-30, $kui-border-radius-30);
+  box-shadow: var(--kui-shadow-border, $kui-shadow-border);
+  color: var(--kui-color-text, $kui-color-text);
+  font-family: var(--kui-font-family-text, $kui-font-family-text);
+  font-size: 16px; // needs to be at least 16px to prevent automatic zoom in on focus on mobile
```

Comment:
suggestion: I still think we should allow customizing this (for non-Kong users); can we use the token?

Reply:
Done.